### ISSUE TITLE
Not working 

### ISSUE THREAD

**Issue by vishnutiwari-tech** [imlakshay08/Waste-Management-System#7]
Both pages(admin and user login) not working

**Comment by imlakshay08** [comment]
On your locally based setup or you are looking on my hosted website? Actually the issue is that my hosting subscription is over and 000webhost just supports static web pages for free, and this could be the reason you are facing this, I am learning Azure right now, and will host this using that.

But if your locally based setupt not working then, it might be due to xampp not configured correctly on your system.

**Comment by vishnutiwari-tech** [comment]
Yes it is locally based setup...what is the right configuration for xampp

**Comment by imlakshay08** [comment]
It is different for every system accordingly, but mostly, you should check that SQL file (wms.sql) is imported correctly.

**Comment by imlakshay08** [comment]
Also, checkout connection.php, the fields should be same according to your Xampp MySQL server's details.

**Comment by vishnutiwari-tech** [comment]
When I'm trying to login it is saying that access denied for user 'root'@'localhost'

**Comment by imlakshay08** [comment]
You might have setup password for accessing your database(PHPMyAdmin) when installing xampp. You need to provide that password so that it could access it.
